Common/UnitDefinitions/Pressure.json

```diff
@@ -365,6 +365,18 @@
         }
       ]
     },
+    {
+      "SingularName": "MeterOfWaterColumn",
+      "PluralName": "MeterOfWaterColumn",
+      "FromUnitToBaseFunc": "{x} * 9.806650000000272e3",
+      "FromBaseToUnitFunc": "{x} / 9.806650000000272e3",
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "mH2O", "m wc" ]
```

Comment:
Maybe add `mH₂O` to start of array? The first is used by default in `ToString()`.

Reply:
It also seems `m wg` is used

> metre water (mH2O), metre water column (m wc) or metre water gauge (m wg)
https://en.wikipedia.org/wiki/Centimetre_or_millimetre_of_water

Reply:
Fixed.